src/App.cpp

```diff
@@ -228,7 +228,7 @@ int MyApp::OnExit()
         m_luaLog->Close();
         // Check to see if we have lua messages
         if (m_luaMessages) {
-            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), (const wxChar *)GetLuaLogFilename().c_str());
+            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), GetLuaLogFilename().c_str().AsChar());

```

Comment:
Yeah . . . that less ideal, but fine and will protect against wx changing things up in the future.  There are only a few canned messages that even use take varargs, and all of them only use one arg.

My initial thought is that they should be templates (since one of the canned messages uses "%d"), but that might complicate things a little.  Just taking wxString means we can pass whatever string we want into the function and conversion will fix it, but a function template would pass the arg untouched.

However, looking through wx code, it appears that their string format templates handle the conversion:

[include/wx/strvararg.h](https://github.com/wxWidgets/wxWidgets/blob/977a8266398a0851a13a640bce90d895d3756a43/include/wx/strvararg.h)
```
// Converts an argument passed to wxPrint etc. into standard form expected,
// by wxXXX functions, e.g. all strings (wxString, char*, wchar_t*) are
// converted into wchar_t* or char* depending on the build.
template<typename T> struct wxArgNormalizer
```